### ISSUE TITLE
came back after an overnight and when i pulled Sapphire out of the dock got this:

### ISSUE THREAD

**Issue by dmf3030** [project-ecc/Sapphire#104]
Sapphire Auto generated github issue.

STACK TRACE:

NonError: { message: "Cannot read property 'getWrappedInstance' of null" }
    at module.exports (/Applications/Sapphire.app/Contents/Resources/app.asar/node_modules/ensure-error/index.js:14:10)
    at handleError (/Applications/Sapphire.app/Contents/Resources/app.asar/node_modules/electron-unhandled/index.js:24:10)
    at /Applications/Sapphire.app/Contents/Resources/app.asar/node_modules/electron-unhandled/index.js:91:4
    at invokeFunc (/Applications/Sapphire.app/Contents/Resources/app.asar/node_modules/lodash.debounce/index.js:160:19)
    at trailingEdge (/Applications/Sapphire.app/Contents/Resources/app.asar/node_modules/lodash.debounce/index.js:207:14)
    at timerExpired (/Applications/Sapphire.app/Contents/Resources/app.asar/node_modules/lodash.debounce/index.js:195:14)

Node Environment: production

Running Environments: 
Sapphire 1.3.0
Electron 6.0.12
darwin 18.6.0
Locale: en-US

**Comment by dmf3030** [comment]
Also I couldn't get the sapphire window to show up, but it wasnt crashed, so I found the top bar and went fullscreen, then it showed up and then I could go back to  a regular sized window.  That error popped again when I did it.  However I'm still connected and synced and running so looks like its possibly something related to minimizing or some kind of dock behavior, that or they are two different issues.
